app/controllers/tasks_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
   def update
     task.assign_attributes(task_params(task.class))
     task.save!
+    task.send_emails if task.respond_to? :send_emails
```

Comment:
Let's implement a no-op `Task#send_emails` method so we can remove this `if` condition. Subclasses can override it.